kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/BrokerConfigsResourceIntegrationTest.java

```diff
@@ -206,6 +206,121 @@ public void getBrokerConfig_nonExistingCluster_throwsNotFound() {
     Assert.assertEquals(Status.NOT_FOUND.getStatusCode(), response.getStatus());
   }
 
+  @Test
+  public void
+  getThenUpdateThenGetThenResetThenGet_existingConfig_returnsDefaultThenUpdatesThenReturnsUpdatedThenReturnsDefault
```

Comment:
nit - Do we modify the test name to something like `getUpdateReset_withExistingConfig` or `compositeBrokerConfigRequests_withExistingConfig` and comments in the test like - ` // call to get broker configs` , ` // call to update broker configs` ...

Reply:
`getUpdateReset_withExistingConfig` sounds good.